README.md

```diff
@@ -1,30 +1,137 @@
 # spanner-sql-udf
 
-This repository contains a set of Spanner User-Defined Functions (UDFs) for a
-number of MySQL built-in functions. Combined together with Spanner's existing
-built-in functions, they provide coverage for most of MySQL's built-in
-functions. These UDFs are designed to assist customers migrating applications
-from MySQL to Spanner by providing definitions for MySQL built-in functions not
-already supported in Spanner, reducing the effort to migrate application code
-and queries.
+This repository contains a package of Spanner User-Defined Functions (UDFs) for
+many MySQL built-in functions. Combined with Spanner's existing built-in
+functions, these UDFs provide coverage for most of MySQL's built-in functions.
+When you migrate from MySQL to Spanner, you can use these UDFs to use MySQL
+built-in functions that Spanner doesn't support. This reduces the effort to
+migrate your application code and queries.
 
-Spanner’s support for UDFs will be available soon.
+## Features
 
-## Features & Limitations
-
-Supported list of User-Defined Functions (UDFs)
+Spanner supports the following UDFs:
 
+- BIN_TO_UUID
+- BIT_LENGTH
+- CHAR
+- CONCAT_WS
+- DATE_FORMAT
+- DATEDIFF
 - DAY
 - DAYNAME
 - DAYOFMONTH
 - DAYOFWEEK
 - DAYOFYEAR
+- DEGREES
+- FROM_DAYS
+- FROM_UNIXTIME
+- HEX
+- HOUR
+- INET6_ATON
+- INET6_NTOA
+- INET_ATON
+- INET_NTOA
+- INSERT
+- IS_IPV4
+- IS_IPV4_COMPAT
+- IS_IPV4_MAPPED
+- IS_IPV6
+- IS_UUID
+- JSON_QUOTE
+- JSON_UNQUOTE
+- LOCALTIME
+- LOCALTIMESTAMP
+- LOCATE
+- LOG2
+- MAKEDATE
+- MICROSECOND
+- MID
+- MINUTE
+- MONTH
+- MONTHNAME
+- NOW
+- OCT
+- ORD
+- PERIOD_ADD
+- PERIOD_DIFF
+- PI
+- POSITION
+- QUARTER
+- QUOTE
+- RADIANS
+- REGEXP_LIKE
+- REGEXP_SUBSTR
+- SECOND
+- SHA2
+- SPACE
+- STRCMP
+- STR_TO_DATE
+- SUBSTRING_INDEX
+- SYSDATE
+- TIME
+- TO_DAYS
+- TO_SECONDS
+- TRUNCATE
+- UNHEX
+- UNIX_TIMESTAMP
+- UTC_DATE
+- UTC_TIMESTAMP
+- UUID
+- UUID_TO_BIN
+- WEEK
+- WEEKDAY
+- WEEKOFYEAR
+- YEAR
+
+## Install the package
+
+-   The package consists of a batch of DDL statements that creates a MySQL
+    namespace and defines UDF functions for many MySQL built-in functions.
+-   Run this batch of DDL statements as you would make any other schema update.
+    For example, you can cut-and-paste the
+    [mysql_udfs.sql](https://github.com/googleapis/spanner-sql-udf/blob/main/mysql/mysql_udfs.sql)
+    file into the Spanner Studio page and click Run.
+
+## Customize the package
+
+-   If you know which functions you'll need, Spanner recommends customizing the
```

Comment:
I think "we recommend" is more our style in documentation -- it shows up a lot in our documentation.
"Spanner recommends" is kind of awkward because Spanner is the database system itself -- although I do see we use it a couple of times in our documentation.